data_preprocessing/main.py

```diff
@@ -34,6 +37,18 @@ def main():
     data_processor.split_data_to_training_and_test()
     data_processor.save_data('../data')
 
+    # Naive Bayes Classifier
+    nb_classifier = NaiveBayesClassifier(data_processor.training_data, data_processor.test_data)
+    nb_classifier.train_and_evaluate()
+
+    # Decision Tree Classifier
+    dt_classifier = DecisionTreeClassifierModel(data_processor.training_data, data_processor.test_data)
+    dt_classifier.train_and_evaluate()
+
+    # K Neighbors Classifier
+    kn_classifier = KNeighborsClassifierModel(data_processor.training_data, data_processor.test_data)
+    kn_classifier.train_and_evaluate()
+
```

Comment:
Personally, I will put this code and all related to classifiers's code into separate folder/module. 
It's better to keep `data_processing` module to do one thing, prepare a data, because we don't want to preprocess data each time we run training

Reply:
Done